components/Workspaces/TrackedRepoWizard/PasteReposStep.tsx

```diff
@@ -9,13 +9,20 @@ export const PasteReposStep = ({ onBulkAddRepos }: PasteReposStepProps) => {
   const [pastedInput, setPastedInput] = useState("");
 
   const parseInput = () => {
-    // split each line into a trimmed string and filter out any empty lines
     const repos = pastedInput
-      .split(pastedInput.includes(",") ? "," : "\n")
-      .map((line) => line.trim())
-      .filter((line) => line !== "");
+      .split(/[,\n ]/g) // split by either comma, new line, or space
+      .map((line) => {
+        line.trim(); // trim to remove whitespace
 
-    onBulkAddRepos(repos);
+        // only take the 'org/repo' from URL, otherwise use the current line
+        const { repo } = /[https:\/\/]?github.com\/(?<repo>[^\/]+\/[^\/]+)/gm.exec(line)?.groups || { repo: line };
```

Comment:
Nice 😎 moving the `https://` into an optional group in the regex is a great way to do this.